__tests__/bitmap.test.js

```diff
@@ -33,10 +33,4 @@ describe('Transforming a bitmap', () => {
     bitmap.transform('flipH');
     expect(bitmap.newFile).toBeDefined();
   });
-
-  it('should have a different buffer than the original', async () => {
-    let transformedBuffer = await fsReadFile(`${__dirname}/../assets/baldy.flipH.bmp`);
-
-    expect(transformedBuffer).not.toStrictEqual(bitmap.buffer);
```

Comment:
So the problem you're running into here is that transforms modify buffers **in place**. If you want to confirm `buffer` changed, you could copy part or all of into a separate Buffer before your test, then `expect(bitmap.buffer).not.toStrictEqual(copyOfOriginalBuffer)`.